kodilla-patterns/src/test/java/com/kodilla/patterns/prototype/library/LibraryTestSuite.java

```diff
@@ -0,0 +1,51 @@
+package com.kodilla.patterns.prototype.library;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.time.LocalDate;
+import java.util.stream.IntStream;
+
+public class LibraryTestSuite {
+
+    @Test
+    public void testGetBooks() {
+        //Given
+        Library library = new Library("Local library");
+        //creating books
+        IntStream.iterate(1, n -> n + 1)
+                .limit(10)
+                .forEach(n -> library.getBooks().add(new Book("Book number" + n,
+                        "Author " + n * 10,
+                        LocalDate.of(2019, n, n * 2))));
+
+        //creating a shallow copy of library
+        Library shallowClonedLibrary = null;
+        try {
+            shallowClonedLibrary = library.shallowCopy();
+            shallowClonedLibrary.setName("City library");
+        } catch (CloneNotSupportedException e) {
```

Comment:
try-catche w testach to zla praktyka. Lepiej przerzuc exception do deklaracji testu
`public void testGetBooks() throws CloneNotSupportedException`